lib/screens/config/v2/pre_fare.ex

```diff
@@ -6,13 +6,15 @@ defmodule Screens.Config.V2.PreFare do
 
   @type t :: %__MODULE__{
           header: CurrentStopId.t(),
+          alert_widget: CurrentStopId.t(),
```

Comment:
Depending on your answer to my question above, should change these to include `reconstructed`.